### ISSUE TITLE
you have Dense example but will it work for sparse data ?

### ISSUE THREAD

**Issue by Sandy4321** [jamborta/tffm-tf2#2]
you have Dense example but will it work for sparse data ?

it is written Dense example in 
https://github.com/jamborta/tffm2/blob/master/example.ipynb

**Comment by jamborta** [comment]
@Sandy4321 yes, it works with sparse data, too. I can add an example if that's useful.

**Comment by Sandy4321** [comment]
yes it would be great to have example for sparse data and both second order and third order code run
for classification and regression tasks  
for example for this data for regression 
https://www.kaggle.com/c/tabular-playground-series-feb-2021
or only data sets with all categorical data may be used, not mixture categorical and continues ?

**Comment by ZiweiHou** [comment]
Hi @jamborta, I'm new to python. just wondering how it works with sparse data as the dense input is nd array while sparse data is in sparse matrix type? However, does the current code accept sparse matrix type as input?
